### ISSUE TITLE
Full freeze of font-manager

### ISSUE THREAD

**Issue by 0rphee** [FontManager/font-manager#344]
**Describe the bug**
When opening the program, it freezes up, without being able to use it at all. This seems to happen in the 'characters' section, when i installed it, it defaulted to the waterfall section, and it was slow with color fonts. However, when changing to the characters section, it froze up. After `^C`'ing it, and starting it again, it froze up from the beginning with the 'Android Arabic Kufi' font preselected. I disabled it with font-manager -d, and when starting it again it defaulted to the font detailed in the following video.

**To Reproduce**
1. Open font-manager
2. Click on 'characters'
3. It freezes up

**Expected behavior**
It doesn't freezes

**Screenshots**
https://github.com/FontManager/font-manager/assets/79347623/6ead2f5c-e1af-47df-81f3-7eed54d3c296

**System Information**
 - OS:Fedora Linux Asahi Remix 39 (Thirty Nine) aarch64 
 - Desktop Environment: Hyprland
 - Font Manager version: font-manager 0.8.8 from latest commit (f6edfc0)
 - Font Manager installation method: source

**Additional context**

**Comment by JerryCasiano** [comment]
Sorry.

I run Fedora but I can't reproduce this. So unfortunately there isn't much I can do about it.

Since no other Fedora users seem to behaving this issue I'm thinking it may be something specific to your particular setup.

**Comment by AtomicRobotMan0101** [comment]
I experienced the same.

No appreciable output (no logs, no disk, no CPU)... simply an apparent freeze.

I left it for ~2 minutes and it came good. I suspect its a font I've installed it doesn't like ingesting....

**Comment by JerryCasiano** [comment]
@AtomicRobotMan0101 

Very likely. unfortunately I'm struggling to come up with a good way to work around this....

If possible, attaching an archive of the files active on your system might be helpful.

**Comment by AtomicRobotMan0101** [comment]
I removed the Fira Code, Jetbrains Mono, Noto mono+sans and the Ubuntu fonts (I use Mint cinnamon 22.3) and installed fresh copies of the Nerd Fonts of the same series.

Problem disappeared.

I don't have the offending font installed or available any longer. Damned shame.

The problem, I am pleased to report, evaporated with the 5 above being removed and installation of the other substitutes. **_WHICH_** of those that caused the "hang" is a mystery. I really should have segmented each off and tested, but didn't consider it at the time. :(

My guess? Either the FiraCode or JetBrains was the issue. BUT, there were no logs, nor exceptions thrown, even when I launched FM from the command line. There were zero logs in anything I could find. It was weird.

All fonts were gathered from the usual sources.

....

Not that it will be any use, but Ive [attached a Fzxxx INXI](https://termbin.com/a7yq) JIC it may be. :)
